hugegraph-api/src/main/java/org/apache/hugegraph/api/cypher/CypherClient.java

```diff
@@ -62,14 +68,14 @@ public CypherModel submitQuery(String cypherQuery,@Nullable Map<String, String>
         }
 
         RequestMessage request = createRequest(cypherQuery);
-        CypherModel res = null;
+        CypherModel res;
 
         try {
             List<Object> list = this.doQueryList(client, request);
             res = CypherModel.dataOf(request.getRequestId().toString(), list);
         } catch (Exception e) {
-            LOG.error(String.format("Failed to submit cypher-query: [ %s ], cause by:"
-                    , cypherQuery), e);
+            LOG.error(String.format("Failed to submit cypher-query: [ %s ], cause by:",
```

Comment:
1. please check all the `String.format`, use "{}" instead
2. `cause by` => `caused by`

Reply:
It's an error message, there isn't a method that allows me to use {} and print the stack trace at the same time.